backend/models.py

```diff
@@ -1,7 +1,12 @@
 from flask_login import UserMixin
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
```

Comment:
is this available as db.relationship ? If so use that.

Reply:
Also check out the documentation from the library you are using. https://flask-sqlalchemy.palletsprojects.com/en/3.1.x/quickstart/#define-models